xwiki-rendering-macros/xwiki-rendering-macro-message/src/main/java/org/xwiki/rendering/internal/macro/message/AbstractMessageMacro.java

```diff
@@ -19,18 +19,22 @@
  */
 package org.xwiki.rendering.internal.macro.message;
 
-import java.util.Collections;
-import java.util.List;
-
 import org.xwiki.rendering.block.Block;
+import org.xwiki.rendering.block.GroupBlock;
 import org.xwiki.rendering.block.MacroBlock;
 import org.xwiki.rendering.block.MetaDataBlock;
+import org.xwiki.rendering.block.match.AnyBlockMatcher;
 import org.xwiki.rendering.macro.MacroExecutionException;
 import org.xwiki.rendering.macro.MacroPreparationException;
 import org.xwiki.rendering.macro.box.AbstractBoxMacro;
 import org.xwiki.rendering.macro.box.BoxMacroParameters;
 import org.xwiki.rendering.macro.descriptor.DefaultContentDescriptor;
 import org.xwiki.rendering.transformation.MacroTransformationContext;
+import org.xwiki.rendering.transformation.icon.IconProvider;
+
+import javax.inject.Inject;
+import java.util.Collections;
+import java.util.List;
```

Comment:
Wrong import order (see also below).

Reply:
Addressed in https://github.com/xwiki/xwiki-rendering/pull/286/commits/95bae640b98f40f32cde3abe4216d3e6c7f14a04 :+1: